src/types/json.rs

```diff
@@ -211,6 +211,42 @@ where
 ///
 /// # Examples
 ///
+/// Global extractor configuration:
+/// ```rust
+/// use actix_web::{error, web, App, FromRequest, HttpResponse};
+/// use serde_derive::Deserialize;
+///
+/// #[derive(Deserialize)]
+/// struct Info {
+///     username: String,
+/// }
+///
+/// /// deserialize `Info` from request's body, max payload size is 4kb
+/// async fn index(info: web::Json<Info>) -> String {
+///     format!("Welcome {}!", info.username)
+/// }
+///
+/// fn main() {
```

Comment:
`fn main()` is not required in doc-tests (unless it's `async`).

Reply:
These are example for the users.  Shouldn't it show `main()`?

Reply:
Well, the `main` function doesn't contain any logic aside of creating an `App`.
I believe it's a matter of taste, but I think that `main` function should not be shown unless it provides more context / meaning for the user.

Reply:
The other doc examples still include the `main` function, so I will keep it for now.